ts/features/fastLogin/store/reducers/index.ts

```diff
@@ -10,15 +10,21 @@ import {
   SecurityAdviceAcknowledgedState,
   securityAdviceAcknowledgedPersistor
 } from "./securityAdviceReducer";
+import {
+  fastLoginSessionRefreshPersistor,
+  FastLoginSessionRefreshState
+} from "./sessionRefreshReducer";
 
 export type FastLoginState = {
   optIn: FastLoginOptInState & PersistPartial;
+  sessionRefresh: FastLoginSessionRefreshState & PersistPartial;
   tokenRefreshHandler: FastLoginTokenRefreshState;
   securityAdviceAcknowledged: SecurityAdviceAcknowledgedState & PersistPartial;
 };
 
 export const fastLoginReducer = combineReducers<FastLoginState, Action>({
   optIn: fastLoginOptInPersistor,
+  sessionRefresh: fastLoginSessionRefreshPersistor,
```

Comment:
What about giving to this property or to its child one a more meaningful name, specifying that once enabled, it drives the automatic refresh during the identification modal showing after an app inactivity of more than 2 min? What do you think?

Reply:
Do you have any ideas for a more descriptive name? I used this name and added comments all over the code because I could not find a more descriptive name 😅

Reply:
Sometimes a long name is better than a misleading or prone-to-interpretations one. Actually I don't have thought about a name, but I can suggest one:
- `automaticSessionRefreshHandling`
  - `afterAppInactivity`: true/false
 ore something like that. 
 
 Feel free to take it only as a reference.

Reply:
I have changed the names of the variables so that they are more descriptive but not too long and I have tried to be more descriptive with the name of the dispatched action. Let me know what you think